data/trie/trieStorageManager.go

```diff
@@ -18,8 +18,8 @@ import (
 )
 
 const (
-	snapshotBufferLen = 10
-	pruningBufferLen  = 100
+	snapshotBufferLen = 100
```

Comment:
if there are more than 100 smart contracts this might be problematic. I think these bufferLens should be not limited, or be super big in terms of millions ?

Reply:
This is a limitation for the buffered channel. If there are more than 100 smart contracts, all will be snapshoted, but the operation will be buffered, meaning that the first 100 will be added to the channel, then when one snapshot is done, another snapshot will be added.